pkg/sql/schemachanger/schemachanger_test.go

```diff
@@ -913,13 +913,33 @@ func TestComparatorFromLogicTests(t *testing.T) {
 	for _, entry := range corpus.Entries {
 		subtestName := entry.Name
 		subtestStatements := entry.Statements
-		t.Run(entry.Name, func(t *testing.T) {
+
+		if shouldSkipLogicTestCorpusEntry(subtestName) {
+			continue
```

Comment:
Would it be helpful to still call `t.Run` and then `t.Skip` for these cases to show that they're captured correctly?

If we don't want to generate skipped tests, maybe just a `t.Log` that we're skipping this?

Reply:
I actually liked the idea to still call `t.Run` and skip inside it! I will change it